src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -423,12 +429,12 @@ static void TransformationUpToCPUSpecificOpSet(const std::shared_ptr<ngraph::Fun
             return !MKLDNNFakeQuantizeNode::isSupportedOperation(node, errMsg);
         });
 
-        pass_config->set_callback<ngraph::pass::ConvertQuantizeDequantize>([](const_node_ptr &node) -> bool {
-            return ngraph::pass::low_precision::NetworkHelper::areQuantizeAndDequantizeSupportedForMultiply(node);
+        pass_config->set_callback<ngraph::pass::ConvertQuantizeDequantize>([defaultPrecisions](const_node_ptr &node) -> bool {
```

Comment:
Capture `defaultPrecisions` by reference
```suggestion
        pass_config->set_callback<ngraph::pass::ConvertQuantizeDequantize>([&defaultPrecisions](const_node_ptr &node) -> bool {
```

Reply:
done, thank you.